rust-toolchain

```diff
@@ -0,0 +1 @@
+1.32.0
```

Comment:
Again, a comment pointing to the `update-rust-version` script would probably be nice (if `rust-toolchain` supports comments? it may not, now that I think of it...)

Reply:
from the docs, and the code I saw doesn't seem to be possible

Reply:
👍 okay, nevermind, then! thanks for looking into it!